assets/js/modules/pagespeed-insights/components/common/LabReportMetrics.js

```diff
@@ -25,18 +25,44 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
+import { useCallback } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
+import API from 'googlesitekit-api';
+import Data from 'googlesitekit-data';
+import { STORE_NAME as CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
 import ReportMetric from './ReportMetric';
 import ReportDetailsLink from './ReportDetailsLink';
 import MetricsLearnMoreLink from './MetricsLearnMoreLink';
 import { getScoreCategory } from '../../util';
+import Button from '../../../../components/button';
 import ErrorText from '../../../../components/error-text';
+import {
+	STORE_NAME,
+	STRATEGY_MOBILE,
+	STRATEGY_DESKTOP,
+} from '../../datastore/constants';
+
+const { useSelect, useDispatch } = Data;
 
 export default function LabReportMetrics( { data, error } ) {
+	const referenceURL = useSelect( ( select ) => select( CORE_SITE ).getCurrentReferenceURL() );
+
+	const { invalidateResolution } = useDispatch( STORE_NAME );
+	const updateReport = useCallback( async ( event ) => {
+		event.preventDefault();
+
+		// Invalidate the PageSpeed API request caches.
+		await API.invalidateCache( 'modules', 'pagespeed' );
```

Comment:
The second parameter here should be the module slug. `pagespeed` is actually the datapoint name for the report selector, which we want to keep. See the control for `getReport` in `assets/js/modules/pagespeed-insights/datastore/report.js`
```suggestion
		await API.invalidateCache( 'modules', 'pagespeed-insights', 'pagespeed' );
```